frontend/packages/dev-console/src/components/pipelines/pipeline-topology/SpacerNode.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { observer, Node } from '@console/topology';
+
+const SpacerNode: React.FC<{ element: Node }> = () => {
+  return <g />;
+};
```

Comment:
Dagre trickery?

Reply:
To get the design we want, yeah... parallel to parallel is just straight lines to Dagre... placing a node as a focal point allows them to flow to an invisible point and then flow back out.